client/main.lua

```diff
@@ -138,15 +116,16 @@ local function CreateZone(type, garage, index)
                 CreateZone("house", garage, index)
                 HouseMarkers = true
             elseif type == "house" then
-                if IsPedInAnyVehicle(PlayerPedId(), false) then
+                if IsPedInAnyVehicle(cache.ped, false) then
```

Comment:
Instead of doing

`if IsPedInAnyVehicle(cache.ped, false) then`

do

`if cache.vehicle then`